.github/workflows/R-CMD-check.yaml

```diff
@@ -44,10 +36,10 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@v2
 
-      - name: Set up Rust
-        uses: dtolnay/rust-toolchain@master
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
what difference did this have?

Reply:
No difference, I just decided that since we have all other steps unnamed, there is no particular reason to keep an exclusive name for rust installation step.